docs/development/development-database.rst

```diff
@@ -0,0 +1,32 @@
+Updating Development Database
+=============================
+
+Occassionally, the database dump supplied for local development should be
+refreshed.
+
+This dump is created from the database behind test.pypi.org, not prodution.
+
+Use ``pg_dump`` to create the initial file::
+
+    $ pg_dump --no-owner $DATABASE_URL > dev/example.sql
+
+Now create a database locally and load the result::
+
+    $ createdb warehouse_dev_dump
+    $ psql warehouse_dev_dump < dev/example.sql
+
+With the database loaded, run the clean script to remove personal information::
+
+    $ psql warehouse_dev_dump < dev/clean.sql
+
+Dump the result, compress it, and clean up::
+
+    $ pg_dump --no-owner warehouse_dev_dump > dev/example.sql
+    $ $ xz -z -9 dev/example.sql
```

Comment:
Double `$` here

Reply:
![make-it-rain-dollars](https://user-images.githubusercontent.com/1200832/41424763-fd5103fa-6fcc-11e8-9e0c-bbeb54f46568.gif)